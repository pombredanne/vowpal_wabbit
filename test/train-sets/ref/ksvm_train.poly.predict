0.000000
0.474998
0.030951
-0.226592
-0.388124
-0.466238
-0.140597
-0.250019
-0.474723
-0.552539
-0.336431
-0.456513
-0.429918
-0.559777
-0.588908
-0.304422
-0.135355
-0.237016
-0.395260
-0.283754
-0.491757
-0.206128
-0.023145
-0.290619
-0.187387
-0.335723
-0.471053
-0.636850
-0.169232
0.001348
-0.378281
-0.160355
-0.355598
-0.442723
-0.282599
-0.291815
-0.416944
-0.284094
-0.092154
0.261195
0.070485
-0.205946
-0.183640
-0.207024
-0.577064
-0.585207
-0.157772
-0.417899
-0.570161
-0.218465
-0.478127
-0.191397
0.006120
-0.205384
-0.074444
0.160806
-0.444292
-0.331456
-0.328921
-0.246170
-0.335503
-0.216857
0.016338
-0.705444
-0.244308
-0.307760
-0.376750
-0.310994
-0.545145
-0.274580
0.016174
0.006337
-0.098968
-0.188930
0.022702
-0.003774
0.081326
-0.524803
-0.456538
-0.489830
-0.299190
-0.343117
-0.090118
-0.245352
-0.557035
-0.515937
-0.520374
0.034592
0.192448
-0.427087
-0.266062
-0.355511
0.439567
-0.176150
0.139074
-0.593854
-0.200058
-0.243954
-0.335328
-0.115632
0.155844
-0.157714
0.073955
-0.026952
-0.019477
0.028067
-0.207354
-0.397144
-0.429748
-0.154303
0.082935
0.108425
-0.312798
-0.437087
0.385775
-0.348345
-0.472655
0.105535
0.094017
-0.077899
-0.214098
-0.156984
-0.234381
-0.286154
0.246932
0.319977
0.003099
0.195576
-0.436369
-0.282076
0.103906
-0.084867
-0.156334
-0.250341
-0.174189
-0.472940
-0.096030
0.075632
0.737474
-0.109968
-0.367205
-0.464433
0.547752
0.800058
-0.175462
0.378849
0.029723
0.791179
-0.529997
-0.212466
0.285627
-0.090211
0.463137
-0.181917
-0.418259
0.133803
0.642412
-0.057123
0.618328
-0.087959
0.684487
-0.427589
-0.440116
0.052760
-0.113725
-0.075335
0.553443
-0.195033
-0.235304
0.558652
0.415538
-0.576421
-0.282549
0.151509
-0.571037
-0.003362
-0.721901
0.705728
0.205467
0.578626
0.229570
0.164604
0.047364
-0.337177
-0.229484
-0.371186
0.094726
-0.416198
-0.189865
0.239079
0.439249
-0.019407
-0.333159
-0.323632
-0.461697
0.008897
0.128423
-0.215056
-0.058592
0.046720
-0.056684
0.305272
0.270415
-0.366464
-0.401122
-0.491095
0.515909
0.109752
0.004363
0.105744
0.116108
-0.644276
-0.424929
-0.050942
1.154799
-0.703287
-0.322689
-0.229192
-0.193624
-0.191527
-0.025285
-0.497856
-0.388862
-0.390558
0.106342
-0.074199
1.246416
0.084491
-0.373468
-0.380989
-0.689688
-0.558624
-0.538119
0.026677
0.133920
-0.431125
-0.401777
0.184638
0.947166
0.098598
0.292114
-0.154739
-0.687989
0.617385
-0.113036
0.186376
0.652234
-0.319503
-0.518012
0.837314
