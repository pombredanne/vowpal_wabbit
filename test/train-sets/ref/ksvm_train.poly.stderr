using l2 regularization = 1
predictions = ksvm_train.poly.predict
Lambda = 1
Kernel = poly
degree = 2
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/rcv1_smaller.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
1.000000 1.000000            1            1.0   1.0000   0.0000       50
1.237499 1.474998            2            2.0  -1.0000   0.4750      103
1.069839 0.902179            4            4.0  -1.0000  -0.2266      134
0.995857 0.921875            8            8.0  -1.0000  -0.2500      145
0.947210 0.898563           16           16.0   1.0000  -0.3044       23
0.940548 0.933886           32           32.0  -1.0000  -0.1604       31
0.924417 0.908287           64           64.0  -1.0000  -0.7054       60
0.913284 0.902150          128          128.0   1.0000   0.1956      105

finished run
number of examples = 250
weighted example sum = 250.000000
weighted label sum = -22.000000
average loss = 0.821119
best constant = -0.088000
best constant's loss = 0.992256
total feature number = 19870
Num support = 247
Number of kernel evaluations = 142949 Number of cache queries = 228978
Total loss = 205.279877
Done freeing model
Done freeing kernel params
Done with finish 
